tests/test_utils_errors.py

```diff
@@ -11,117 +11,117 @@
     RepositoryNotFoundError,
     RevisionNotFoundError,
 )
-from huggingface_hub.utils._errors import REPO_API_REGEX, hf_raise_for_status
+from huggingface_hub.utils._http import REPO_API_REGEX, X_REQUEST_ID, _format, hf_raise_for_status
 
 
 class TestErrorUtils(unittest.TestCase):
     def test_hf_raise_for_status_repo_not_found(self) -> None:
         response = Response()
-        response.headers = {"X-Error-Code": "RepoNotFound", "X-Request-Id": 123}
+        response.headers = {"X-Error-Code": "RepoNotFound", X_REQUEST_ID: 123}
         response.status_code = 404
         with self.assertRaisesRegex(RepositoryNotFoundError, "Repository Not Found") as context:
             hf_raise_for_status(response)
 
-        self.assertEqual(context.exception.response.status_code, 404)
-        self.assertIn("Request ID: 123", str(context.exception))
+        assert context.exception.response.status_code == 404
+        assert "Request ID: 123" in str(context.exception)
```

Comment:
Out of curiosity, why the change from `self.assertEqual`/`self.assertIn` to blank asserts with boolean statements? For consistency with new tests?

Reply:
Mostly for consistency yes. I'm only doing it when I'm adding tests next to existing ones (see related [convo on slack](https://huggingface.slack.com/archives/C021H1P1HKR/p1707406613622809?thread_ts=1707404663.451959&cid=C021H1P1HKR) -private-)